events_registry/events_registry.py

```diff
@@ -0,0 +1,43 @@
+import importlib
+import inspect
+import os
+
+FOLDER_PATH = "events"  # TODO: Should it be here, or move to general config?
```

Comment:
I think we can keep it here for now, since events folder doesn't change, and sticks to this repo structure